lib/truthtable.ts

```diff
@@ -483,6 +758,12 @@ interface TruthTable {
     results: { assignment: boolean[]; result: boolean; }[];
 }
 
+/**
+ * Generates a truthtable with the supplied AST & variables
```

Comment:
Should we use abbreviations that may not be known to all developers like ast? or should we use the written out from?